app/services/sessionsManager.js

```diff
@@ -5,10 +5,23 @@ const SECRET = config.common.session.secret;
 
 exports.HEADER_NAME = config.common.session.header_name;
 
+exports.blackList = [];
+
 exports.encode = token => {
   return jwt.encode(token, SECRET);
 };
 
 exports.decode = token => {
   return jwt.decode(token, SECRET);
 };
+
+exports.blacklisted = (email, timeOfLogin) => {
+  const blacklistedUser = this.blackList.find(function(element) {
```

Comment:
what will find this if some user invalidates his sessions more than once? the first or the last entry?